src/services/config.js

```diff
@@ -0,0 +1,103 @@
+const common = require('@constants/common')
+const organizationExtensionsQueries = require('@database/queries/organizationExtensions')
+const _ = require('lodash')
+const entites = require('@database/queries/entities')
+const entityType = require('@database/queries/entityType')
+const httpStatusCode = require('@generics/http-status')
+const responses = require('@helpers/responses')
+const { Op } = require('sequelize')
+module.exports = class configsHelper {
+	/**
+	 * List Configs.
+	 * @method
+	 * @name list
+	 * @param {Integer} organization_id
+	 * @param {String} id -  id.
+	 * @returns {JSON} - List of configs as response.
+	 */
+
+	static async list(organization_id) {
+		try {
+			let orgExtenstionData = {}
+			let configData = []
+			// define filter
+			const filter = {
+				organization_id,
+			}
+			// attributes to fetch from organisation Extenstion
+			const attributes = process.env.INSTANCE_LEVEL_CONFIG_ATTRIBUTES.split(',')
+
+			const entity_type_and_entities = await entityType.findUserEntityTypesAndEntities({
+				organization_id: organization_id,
+				value: 'resources',
+			})
+			// fetch the current list of resources
+			const resourceList = entity_type_and_entities[0].entities
+
+			// convert the object into array
+			const resourceListArr = resourceList.map(({ value }) => value)
+
+			// instance level configurations from env as default configs
+			const default_configs = {
+				review_required: process.env.REVIEW_REQUIRED === 'true' ? true : false,
+				show_reviewer_list: process.env.SHOW_REVIEWER_LIST === 'true' ? true : false,
+				min_approval: Number(process.env.MIN_APPROVAL),
+				review_type:
+					process.env.REVIEW_TYPE.toUpperCase === common.REVIEW_TYPE_SEQUENTIAL
+						? common.REVIEW_TYPE_SEQUENTIAL
+						: common.REVIEW_TYPE_PARALLEL,
+			}
+
+			// fetch the configuration from Organization extension for the user's organization
+			orgExtenstionData = await organizationExtensionsQueries.findMany(filter, attributes)
+
+			// get the list of resource types not set by the org-admin
+			let resourceTypeFromDB = []
+
+			// fetch the config data
+			configData = resourceListArr
+				.map((resourceType) => {
+					const filterData = orgExtenstionData.filter((orgExt) => {
+						if (orgExt.resource_type.toLowerCase() === resourceType.toLowerCase()) {
```

Comment:
@adithyadinesh0412 you don't need to convert to lower case while creating the extension we can convert to lower case